ecosystem/sep-TBD.md

```diff
@@ -0,0 +1,154 @@
+
+Here is an example stellar.toml file describing an asset representing a unique digital image, mimicking the JSON metadata file we described [earlier](#describing-your-nft):
+
+```toml
+[DOCUMENTATION]
+ORG_URL="<https://ink.litemint.store>"
+
+[[CURRENCIES]]
+issuer="GAKZGD5BFXZ7P7P45WHTM6DODMOEWWJUSCAIPGYIVIPYQSVR6MM6YR7M"
+code="DEMOASSET"
+name="A demonstration of NFT metadata"
+desc="This is a description of the cool NFT used in this demo."
+image="https://cloudflare-ipfs.com/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/I/m/RickAstleyNeverGonnaGiveYouUp7InchSingleCover.jpg"
+fixed_number=1
+display_decimals=7
+```
+
+Once you have a `stellar.toml` file under your domain, you should also configure the issuing account's `homedomain` to point to it via the [`SetOptions` operation][set-options]. This unifies the three separate components: the account issuing the NFT, the metadata describing the NFT, and the NFT itself.
+
+#### SEP-1 Extensions
+Since NFTs can represent anything, but the SEP-1 currency specification does not have a generic way to refer to an "anything." There is an `image` key, but it is generally used for something like a currency logo.
+
+Thus, this SEP adds additional supported fields to the SEP-1 currency specification to faciliate this need:
+
+| Field Name | Description and purpose |
+|------------|-------------------------|
+| `url`      | A [valid URL][url] pointing to a NFT that this asset represents |
+| `urlhash`  | A [SHA-256][sha2] hash of the data pointed to by `url` |
+
+The `urlhash` is optional and provided here as a way to verify the integrity of the NFT. It's particularly useful if the NFT lives on a different server relative to the `stellar.toml` file. Some URLs (like IPFS CIDs) have integrity "batteries included" and won't need this field.
+
+
+## Design Rationale
+A key component of a flourishing NFTs marketplace is _interoperability_: that drives all of the design decisions in this informational SEP. The [first section](#minting-nfts) is a set of best practices for a _particular_ set of needs in creating NFTs. There is no "one size fits all" way to do this, so creators should feel free to deviate from the recommendations to fulfill their needs. The [second section](#representing-nfts) adds some extremely flexible fields to SEP-1 to accomodate the fact that NFTs can be anything.
+
+It's worth noting that the idea of "ownership" described throughout the SEP—ownership of an asset and its relationship to owning the respective NFT—is a little diluted: neither the network nor a standard can make any claims about _legitimacy_ of the NFT itself. Owning a unit of the Stellar asset representing your NFT is a way to establish a _relationship_ between buyer and seller that is _linked_ to the NFT, nothing more.
```

Comment:
```suggestion
It's worth noting that the idea of ownership described throughout the SEP—ownership of an asset and its relationship to owning the respective NFT—is a little diluted: neither the network nor a standard can make any claims about _legitimacy_ of the NFT itself. Owning a unit of the Stellar asset representing your NFT is a way to establish a _relationship_ between buyer and seller that is _linked_ to the NFT, nothing more.
```

Reply:
The quotes add emphasis, imo. What's the rationale to drop them?

Reply:
I avoid emphasis in SEPs because it suggests that one statement is more important than another. Usually in the SEPs everything is equally important, if something isn't important we probably shouldn't include it. Quotes are also an interesting way to emphasize because most of the time on a single word I read them like air quotes, which loosens meaning of the word.

I generally agree with this statement:

>The strength of your words should make any formatting unnecessary, but if you really want to emphasize something, use boldface or italics.

Ref: https://leffcommunications.com/2017/03/10/how-to-use-quotation-marks/

(The exception with emphasis is that we've found in some SEPs where there are a significant number of components emphasising with bold component names can help navigate long and confusing flows. Kudos to @JakeUrban for coming up with that.)

Reply:
>  Quotes are also an interesting way to emphasize because most of the time on a single word I read them like air quotes, which loosens meaning of the word.

You hit the nail on the head here: the concept of `ownership` is loosened here from what people are used to (I used the wrong word "emphasis" earlier), and that loosening is qualified by the remainder of the paragraph.

Regardless, I don't feel strongly, was just curious about the rationale. Appreciate the link :+1: